### ISSUE TITLE
@web/polyfills-loader doesn't allow sequential loading.

### ISSUE THREAD

**Issue by cranphin** [modernweb-dev/web#1326]
Code generated by the polyfills-loader current looks like this:
    polyfills.push(loadScript('./polyfills/date-time-format-time-zone-no-data-min.c73fc1f563a0e289a3582f4f5d681b58.js'));
    polyfills.push(loadScript('./polyfills/locale-en-amsterdam.3cda29c9dedd6a829f2a1b5a8fe52e14.js'));
    polyfills.push(loadScript('./polyfills/metazone.fe395f00583be12cfb9f11e6b94f9f71.js'));
...
   Promise.all(polyfills).then(loadFiles);

In most cases it's nice that all loads all scripts in parallel (faster).
But in this example date-time-format-time-zone-no-data-min (from https://github.com/formatjs/date-time-format-timezone) should be loaded before any scripts that add specific time zone data.
It would be nice to have a way to configure that some scripts depend on others, or a more general flag to say all custom scripts should be loaded in sequence. Or a inbetween, a flag you can set on any custom script that must be loaded before the other custom scripts.

**Comment by cranphin** [comment]
Actually, looking at it more it's all a bit complicated.
Behavior wise, and this is IE11, I see error on the console that suggest scripts are not loaded sequentially.
How this works exactly I'm not sure though, loadScript is defined as:
```
  function loadScript(src, type) {
      return new Promise(function (resolve) {
        var script = document.createElement('script');
        function onLoaded() {
          if (script.parentElement) {
            script.parentElement.removeChild(script);
          }
          resolve();
        }
        script.src = src;
        script.onload = onLoaded;

        script.onerror = function () {
          console.error('[polyfills-loader] failed to load: ' + src + ' check the network tab for HTTP status.');
          onLoaded();
        };

        if (type) script.type = type;
        document.head.appendChild(script);
      });
```
So it looks like a new unparented <script> element is created for each polyfill, with it's src set, which is then added to the document head. Not sure which specs would say if those are handled sequentially or simultaniously, especially for IE11 :)

Having one loadScript wait for the previous one (.then) would certainly make it sequential though I guess.

**Comment by cranphin** [comment]
In fact, since the loader uses promises, would/should the loader work for IE11 at all??

**Comment by LarsDenBakker** [comment]
Indeed, we load core-js first and then load all other scripts in parallel. We could look into adding an option for making some groups which are sequential, but it might add more code to the loader script which is not ideal. 

Are these really polyfills? And are they loaded using different conditions? And for which browsers? In some cases it can be simpler to just concatenate them together.
